courses/machine_learning/deepdive2/time_series_prediction/4_modeling_keras.ipynb

```diff
@@ -0,0 +1,1261 @@
+{
```

Comment:
<p>Capitalize TensorFlow, remove extra period a&nbsp;<a href="https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv1D" target="_blank">Conv1d in tensorflow.</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/689/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>done</p>